xbmc/utils/URIUtils.cpp

```diff
@@ -335,10 +335,20 @@ bool URIUtils::GetParentPath(const std::string& strPath, std::string& strParent)
   }
   else if (url.IsProtocol("plugin"))
   {
+    if (!url.GetOptions().empty())
+    {
+      // TODO: make a new python call to get the plugin content type and remove this temporary hack
```

Comment:
Can you annotate this with `@todo` instead? I am not sure doxygen will be happy and it will probably be missed from the list of todos here:
https://codedocs.xyz/xbmc/xbmc/todo.html